pkg/http/rest/api_test.go

```diff
@@ -0,0 +1,81 @@
+package rest
+
+import (
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"net/http"
+	"testing"
+)
+
+var (
+	emptyValue           = ""
+	fakeUrl              = "https://github.com/TurnsCoffeeIntoScripts/jira-api-resource"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var fakeUrl should be fakeURL](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)